README.md

```diff
@@ -1,6 +1,17 @@
 # Kafka
+Apache Kafka is a highly scalable, fast, and fault-tolerant distributed streaming platform. 
+
 This repo publishes the docker image and helm chart for [Apache Kafka](https://kafka.apache.org/).
 
+## How do we use kafka?
+Kafka forms the core of the Hypertrace streaming platform and being used extensively across the services. Kafka usage at Hypertrace can be categorised into 3 categories: Plain vanilla Kafka Clients, Kafka streams and Streaming pipeline jobs. We are migrating streaming pipeline jobs also to Kafka streams.
+We use Confluent Avro schema-registry as a serialization mechanism for the messages published to Kafka and these Schemas are defined in the code along with their respective owner modules.
```

Comment:
link to the actual repo for schema-registry. Also link to the location of the avro files, or "ex XX" if they are indeed spread out to N repos.